CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # `chainweb-node` Changelog
 
+## 2.22 (2023-11-XX)
```

Comment:
Which day are we actually doing the release? Today?

Reply:
We can consider today to be the release cut date, yeah. Shipping is a week out.